src/prefect/flows.py

```diff
@@ -42,7 +42,7 @@
 from prefect.futures import PrefectFuture
 from prefect.logging import get_logger
 from prefect.results import ResultSerializer, ResultStorage
-from prefect.server.schemas.core import raise_on_invalid_name
+from prefect.server.schemas.core import Flow, FlowRun, raise_on_invalid_name
```

Comment:
this shadows the `Flow` class defined below, see https://github.com/PrefectHQ/prefect/issues/8688